### PR TITLE
Look improvements

### PR DESCRIPTION
**Pull request by denysxftr**
screenshot 2015-09-07 17 30 13
screenshot 2015-09-07 17 30 23
screenshot 2015-09-07 17 38 08

### CODE REVIEW

File: app/assets/stylesheets/_components.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+p.issue-desc {
+  font-size: 1.2rem;
+  margin-bottom: 2rem;
+
+  img {
+    max-height: 500px;
+    max-width: 100%;
+  }
+}
```
Comment (tilast) [inline app/assets/stylesheets/_components.scss]: empty line :)

**Comment by tilast** [comment]
looks nice :+1:

**Comment by rilian** [comment]
nice

**Comment by darthrevan** [comment]
PENTAKILL!!!

**Comment by zuzusik** [comment]
IMO, indented lists without list point symbols looks ugly.

I'd totally remove indentation on screen one and added list signs (circles/dashes/etc)

On screen two I'd remove indentation of item description. It is already highlighted by another style of the text

**Comment by zuzusik** [comment]
In other words - let lists to be lists.

**Comment by denysxftr** [comment]
@zuzusik  better?
screenshot 2015-09-07 17 55 11
screenshot 2015-09-07 17 55 17

**Comment by zuzusik** [comment]
@ololyay much better, but still https://github.com/rilian/newsletter/issues/10 is actual

**Comment by denysxftr** [comment]
@zuzusik 
screenshot 2015-09-07 18 00 24

**Comment by zuzusik** [comment]
@ololyay :+1:

**Comment by rilian** [comment]
nice

**Comment by denysxftr** [comment]
#11 #10 #8 #7 #6 are fixed in this PR. And #9 should be discussed.

**Comment by rilian** [comment]
well done :+1:

**Pull request merged**